contracts/upgradeable_contracts/amb_native_to_erc20/HomeAMBNativeToErc20.sol

```diff
@@ -73,7 +72,7 @@ contract HomeAMBNativeToErc20 is BasicAMBNativeToErc20 {
         require(msg.value > 0);
         require(withinLimit(msg.value));
         addTotalSpentPerDay(getCurrentDay(), msg.value);
-        setMediatorBalance(mediatorBalance().add(msg.value));
+        _setMediatorBalance(mediatorBalance().add(msg.value));
```

Comment:
it should be reflected in `callflows.md` similar to other contracts supporting this feature.

Reply:
Added these calls to `amb_native_to_erc20/callflows.md` in 3f8366f